scylla/src/statement/prepared.rs

```diff
@@ -98,30 +98,28 @@ pub struct PreparedStatement {
     /// Tracing IDs of all queries used to prepare this statement.
     pub prepare_tracing_ids: Vec<Uuid>,
```

Comment:
Is this good idea to give full access to `prepare_tracing_ids` ? don't see good reason for giving a write access to it.
Does it make sence to have a getter for it and mark it as depricated, to be removed in next major/minor ?

Reply:
Yes it would be good to move it to shared data, and not have it as public field. I'll add a todo.

Reply:
https://github.com/scylladb/scylla-rust-driver/issues/1443